exopy/tasks/tasks/base_views.enaml

```diff
@@ -88,13 +88,11 @@ enamldef RootTaskView(BaseTaskView): main:
         BaseTaskView.refresh(self)
         editor.refresh()
 
-    # TODO pass the future parent task so that config can do name inspections
-    # and avoid duplicate names.
     func create_new_task():
```

Comment:
Looking at the full diff, I realized I would prefer to follow more closely what I was suggesting in this TODO and pass the future parent rather than the node in case we decide to ever use a different validation or need it for specific config.

Reply:
I don't want to pass te future parent because I want the names to be unique in the entire tree. This makes moving nodes in the tree easier.

Reply:
Ok, makes sense, I am reworking the PR.

Reply:
Ok let me know if you need help with anything.